packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -792,18 +792,22 @@ function emitBasicOperation(
   }
 
   let bodyParameter: any | undefined;
+  // FIXME handle the empty body case
```

Comment:
Are we planning to cover this in the current PR or later work? If later, let's file an issue to track

Reply:
I don't think we don't need this comment more